src/unified_recon/api/models.py

```
@@ -66,3 +67,86 @@ class ErrorResponse(BaseModel):
+    # Exchange group ID to results mapping
+    # e.g., {"1": {...}, "4": {...}}
+    root: Dict[str, ExchangeResult] = Field(
+        ..., description="Position analysis results by exchange group ID"
+    )
+    
+    class Config:
+        """Pydantic config."""
+        
+        json_encoders = {
+            Decimal: lambda v: float(v)
+        }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)